docs/pages/Kratos/Spatial_Containers/kd_tree.md

```diff
@@ -0,0 +1,85 @@
+---
+title: KDTree
+keywords: search spatial_container core
+tags: [search spatial_container]
+sidebar: kratos_core_search
+summary: A k-d tree (short for k-dimensional tree) is a space-partitioning data structure for organizing points in a k-dimensional space.
+---
+
+# KDTree
+
+![A 3-d tree](https://github.com/KratosMultiphysics/Documentation/blob/master/Wiki_files/Search/3dtree.png)
```

Comment:
Small question. I am not sure whther the images will be displayed or a link to images will be displayed. If it is the latter, then I would do it as the following:
https://github.com/KratosMultiphysics/Kratos/blob/e55171cd1820a583298ef16859bf6a9ca6bac685/docs/pages/Applications/Optimization_Application/Filtering/Explicit_Vertex_Morphing.md?plain=1#L13-L16

Reply:
A priori the images should work as I added them

Reply:
good to know for future :)

Reply:
I dont think images are working like that, since the direct link does not give the raw data. :/

Reply:
Yes, you were right

Reply:
So did it make the conversion or not? who was right?

Reply:
@sunethwarna was right